arch/src/x86_64/mpspec.rs

```diff
@@ -1,4 +1,5 @@
 // Copyright 2017 The Chromium OS Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
```

Comment:
Ditto: need to be moved after the existing license information with a blank line in between.

Reply:
done